app/views/discovered_hosts/_discovered_host_modal.html.erb

```diff
@@ -18,13 +17,13 @@
             <% if show_location_tab? %>
                 <%= select_f f, :location_id, Location.my_locations, :id, :to_label, {}, {:size => 'col-md-10'} %>
             <% end %>
-          </div>
-          <div class="modal-footer">
-            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-            <input type="submit" class="btn btn-default" value="<%= _('Quick Create') %>" name="quick_submit" />
-            <input type="submit" class="btn btn-primary" value="<%= _('Create Host') %>" />
-          </div>
-      <% end %>
+        <% end %>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+        <input type="submit" class="btn btn-default" value="<%= _('Quick Create') %>" name="quick_submit" />
+        <input type="submit" class="btn btn-primary" value="<%= _('Customize') %>" />
```

Comment:
I don't really agree with that comment - https://groups.google.com/forum/?fromgroups#!topic/foreman-dev/q_6bxIsEKiU - `Create Host` is more accurate and expected IMO.